security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+For instance, larger payments have fewer routing options.
+This may allow an adversary who controls well-capitalized nodes to route most large payments and discover payment amounts and probably other details.
+Another relevant difference between Lightning and Bitcoin is that Lightning nodes maintain a permanent identity, whereas Bitcoin nodes do not.
+A Bitcoin user can easily switch nodes used to receive blockchain data and broadcast transactions.
+A Lightning user, on the contrary, sends and receives payments through the nodes they have opened channels with.
+Moreover, the Lightning protocol assumes that routing nodes announce their IP address in addition to their node ID.
+This creates a permanent link between node IDs and IP addresses, which may be dangerous, considering that an IP address is often an intermediary step in anonymity attacks linked to the user's physical location and, in most cases, real-world identity.
+It is possible to use Lightning over Tor, but many nodes do not use this functionality.
+
+A Lightning user, when sending a payment, has its neighbors in its anonymity set.
+Specifically, a routing node only knows the immediately preceding and following nodes.
+The routing node does not know whether its immediate neighbors in the payment route are the ultimate sender or receiver.
+Therefore, the anonymity set of a node in Lightning roughly equals its neighbors.
+Similar logic applies to payment receivers.
+Many users open only a handful of payment channels, therefore, limiting their anonymity sets.
+Moreover, in Lightning, the anonymity set is static or at least slowly changing.
+In contrast, one can achieve significantly larger anonymity sets in on-chain CoinJoin transactions.
+CoinJoin transactions with anonymity sets larger than 50 are quite frequent.
+Typically, the anonymity sets in a CoinJoin transaction correspond to a dynamically changing set of users.
+
+[[anonymity-set]]
+.The anonymity set of Alice and Bob constitutes their neighbors.
+image:images/anon-set.png[]
+
+Finally, Lightning users can also be denied service, having their channels blocked by the attacker.
+Forwarding payments requires capital - a scarce resource! - to be temporarily blocked in in-flight HTLCs along the route.
+An attacker may send many payments but fail to finalize them, occupying honest user's capital for long periods.
+This attack vector is not present (or at least not as obvious) in Bitcoin.
+
+In summary, while some aspects of the Lightning Network's architecture suggest that it is a step forward in terms of privacy compared to Bitcoin, other properties of the protocol may make attacks on privacy easier.
+Thorough research is needed to evaluate what privacy guarantees the LN provides and improve the state of affairs.
+We are happy to report that multiple researchers teams are working on Lightning privacy.
+In part, this chapter summarizes the research papers available at the time of writing in late 2020.
```

Comment:
I think this section is missing the clear (imo) advantage privacy wise: you as a user _select_ the set of nodes you wish to interact with in order to send payments, and the transcript of your payments (the partial information) is known only to those nodes. When transacting on-chain, the transaction graph is made public for all of time, and you don't necessarily select which nodes hear about your transactions, or which miners mined the transactions. 

Consider the following statement: so far in 2020, we know _exactly_ how many Bitcoin transactions have happend off-chain. However, no one is able to make a strong claim w.r.t the number of LN transactions that have happened off-chain.